docs/ManualDeploymentGuideScalarDLOnAWS.md

```diff
@@ -69,6 +69,8 @@ Install the following tools on your bastion for controlling the EKS cluster:
 * You must create a Kubernetes cluster with version 1.19 or higher for Scalar DL deployment.
 * You must create a node group with the label, key as `agentpool` and value as `scalardlpool` for Scalar DL deployment.
 * You must add a rule in the EKS security group to **enable HTTPS access (Port 443)** to the private EKS cluster from the bastion server.
+* You must add Taints `kubernetes.io/app=scalardlpool:NoSchedule` to each worker node.
```

Comment:
Manual deployment of Scalar DL worked fine for me without taint.
I did not face any issues related to this.
Let me check more about it and approve it.
Sorry for the delay.